subcommittees/mentoring/README.md

```diff
@@ -1,70 +1,86 @@
 
-### Instructor-Helper Retreat
-In November 2015, Software Carpentry and Data Carpentry instructors and helpers got together together at sites around the world for a day of sharing skills, trying out new lessons and ideas and discussing all things instructor and helper related. 
+### Mentoring Program
 
-To see the schedule for the 2015 retreat, visit: http://swcarpentry.github.io/instructor-retreat-2015/
+Need to add
 
-### Archived: Pre- and Post- Workshop Debriefing Session
+### Coming Soon: Regional Communities Sub-Committee
 
-Before launching the Instructor Discussion Sessions, we used to host separate [pre- and post-workshop debriefing sessions](http://software-carpentry.org/blog/categories/#debriefing). We are grateful to all the hosts and participants who attended these sessions.
+Currently under review, hopefully coming soon!  
 
-## Mailing List
+> The group will facilitate and help those Carpentry members and Member Organizations who wish to  organize and maintain local mentoring groups. This group will also discuss regional community building based on shared missions or topics matching the various interests of regional Carpentry members. 
 
-The best way to contact this subcommittee is via the [mailing
-list][mailing-list], which is archived [here][mailing-list-archives].
+### Mentoring Committee Meetings
 
-## Outreach
+The mentoring committee itself meets regularly to evaluate current activities, and propose
+new activities.  Minutes of past meetings are available [here](minutes).
 
-We [blog][] about our ongoing and proposed activities.  Past blog
-posts are available [here][blog-archives].
+To attend our next meeting, visit this link: http://pad.software-carpentry.org/scf-mentoring
 
-## Meetings
+## How Can I Participate?
 
-We have occasional [meetings][].  Minutes of past meetings are
-available [here](minutes).
+I would like to: 
 
-## Join Us!
+- join a small group to receive mentoring from experienced instructors OR
+- mentor new instructors
+	- Sign up for the mentoring program by contacting Erin Becker.  
+- discuss workshop experiences with other instructors OR
+- ask questions of other instructors as I prepare to teach
+	- Join an instructor discussion session by signing up on this etherpad: http://pad.software-carpentry.org/instructor-discussion
+- help lead discussion sessions among instructors
+	- Contact the discussion coordinators, and sign up to host, co-host, or observe 
+	a discussion on this etherpad: http://pad.software-carpentry.org/instructor-discussion
+- support current mentoring activities or initiate new ones
+	- Sign up for the mentoring committee mailing list and attend the next mentoring committee meeting
+	
+For a summary of different mentoring roles, see [this page](roles.md).  
 
-We welcome new members to the subcommittee.  If you are interested in
-joining our community-building efforts, please email the [mailing
-list](#mailing-list).
+## Committee Members 
 
```

Comment:
The Mentoring Program is currently not accepting new mentors. I have it on my schedule to "reopen" in March. Can you please change this language to reflect that schedule? People who are interested in being Mentors should still contact me, but I'd like them to know ahead of time that we're looking for people to start in March or later.